src/TheHobbit.js

```diff
@@ -0,0 +1,19 @@
+import REACT from 'react';
+import thehobbit from './images/thehobbit.jpg';
+import TheHobbit from './TheHobbit.css';
+
+function Hobbit () {
+    return (
+        <div className = 'principal'>
+            <p id='parrafoprincipal'> The Hobbit, fantasy novel by J.R.R. Tolkien, published in 1937. The novel 
+                introduced Tolkien’s richly imagined world of Middle Earth in its Third 
+                Age and served as a prologue to his The Lord of the Rings.</p>
+            <h2 id='tituloprincipal'> The Hobbit </h2>
+            <h6 id='h66'> novel by Tolkien </h6>
+                <button type="button" id='botoncomprar'> BUY ITEM </button>
+            <img src={thehobbit} id='hobbit'/>
```

Comment:
Si solo pudiera leer tu componente sin ver el resultado en el navegador, esperaría encontrar un párrafo antes que los headings, y la imagen al final. Parte de HTML semántico es también crear una estructura fácil de leer y entender

No es incorrecto agregar estilos a los elementos con ids, pero el uso principal de los ids es para diferenciar un elemento único. En este caso cada elemento era único porque no hay más elementos en la app 😅, pero siempre intenta pensarlo en una escala más grande. Si esto fuera una aplicación más grande probablemente vas a querer re-utilizar las reglas de estilos de los botones, de los títulos, del contenido, imagen, etc. Entonces rule-of-thumb "Siempre usar clases, si el elemento es súper único e irrepetible, _no hay nadie cómo él_, es _Michael Jackson_, es _la última coca del desierto_, entonces usar id"